app-frontend/src/app/pages/browse/browse.component.js

```diff
@@ -0,0 +1,8 @@
+import browserTemplate from './browse.html';
```

Comment:
I think we want to avoid diverging in how we're structuring pages / components at this point, and restructure them all to be better once ui-router 1.0.0 comes out.  That way we just have to work with one type of refactor instead of a couple intermediate formats.